docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -83,7 +83,140 @@ are:
 
 -  A new interface instance can be added to a template.
 
-Please consult the detailed documentation below for more information.
+The following table summarizes the changes supported by that SCU. Consult
+the detailed documentation below for additional information.
+
++-----------------+---------------+------------+-----------------------------------+
+| **Daml Model    | **Scope**     | **Covered  | **Comments**                      |
+| Change**        |               | by SCU?**  |                                   |
++=================+===============+============+===================================+
+| Rename of       | Template      | No         |                                   |
+| Module          | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Rename of       | Template      | No         |                                   |
+| Template        | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field Type      | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field           | Definition    |            |                                   |
+| Ordering        |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field Name      | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | A new template version needs to   |
```

Comment:
Unsure how I feel about marking this as simply "Not covered by SCU". We have special logic to allow these to change with new contracts, and to recompute previous values.
Perhaps "Covered by SCU" isn't the right phrasing here, but instead something akin to "Can change in an upgrade"